pages/common/in2csv.md

```diff
@@ -0,0 +1,20 @@
+# in2csv
+
+> Converts various tabular data formats into CSV.
+> Included in csvkit.
+
+- Convert an XLS file to CSV:
+
+`in2csv {{data.xls}}`
+
+- Convert a DBF file to a CSV file:
+
+`in2csv {{data.dbf}} > {{data.csv}}`
+
+- Convert a specific sheet from an XLSX file to CSV:
+
+`in2csv --sheet={{sheet_name}} {{data.xlsx}}`
+
+- Fetch csvkit's open issues from GitHub's JSON API, and convert them to CSV:
```

Comment:
I'd second that, we have some more like this one. `cat` is nice and basic. It might even be possible to drop the stream to file since a reader should well realize that csv is coming out, ready to be written anywhere.